packages/pluto/readme.md

```diff
@@ -0,0 +1,59 @@
+# Pluto
+
+### An end to end testing framework.
+
+## Setting up for local development
+
+### Clone this repository
```

Comment:
The readme is not user friendly. Assume the person who is going to use it is not working with us. They just want to install this tool and run some tests. Some example tests will help.

Reply:
I'll update it, thanks.